Gaijinizer patch/Map171.txt

```diff
@@ -4,90 +4,97 @@
 
 この居場所まで突き止めるのに、ずいぶん苦労したわよ…
 > CONTEXT: Map171/events/2/pages/0/8/Dialogue/other1
-...... Well, I guess it was the commander class
-of this organization. I had a lot of trouble
-finding this place ...#MTLed
+...To think you'd be the commander of this
+organization. It was a lot of trouble to track
+you down...#Ron
 > END STRING
 
 > BEGIN STRING
 君は………レイナ君、だったか。
 
  相変わらず、『人間』を続けているようだね。
 > CONTEXT: Map171/events/2/pages/0/22/Dialogue/other2
-You are ......... Reina-kun ,?
-It seems that he is continuing "human" as usual.#MTLed
+You're... Reina-kun, yes?
+
```

Comment:
Hmm... This is actually a perfect translation, but I feel like it would be better to avoid contractions with the way the "Commander" speaks.

I'd change "You're" to "you are" here.